src/agent_toolkit/forestadmin/agent_toolkit/services/permissions/permission_service.py

```diff
@@ -197,6 +197,5 @@ async def _find_action_from_endpoint(
             action
             for action in actions
             if action["id"] == f"{collection.name}-{get_params['action_name']}-{get_params['slug']}"
-            and http_method.value == action["httpMethod"]
```

Comment:
This line doesn't exists in agent-nodejs; I could set `and http_method.value == "POST"` but I don't get the value of this verification.